index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hikebuddies - Login</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Mulish:ital,wght@0,200..1000;1,200..1000&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="dist/css/index.css">
+</head>
+<body class="bg-login-body-green-white">
+  <div class="text-center pt-4 d-lg-none">
+    <img src="assets/logo/HikeBuddy-logos_transparent.png" alt="Main logo for HikeBuddy">
+  </div>
+    <div class="container p-0 ">
+    <div class="row rounded-lg-login bg-login-green-white px-0 mt-md-5 frontpage-height">
+      <div class="col-lg-6 d-none d-lg-block p-0 frontpage-height">
+        <img src="assets/img/ivana-cajina-e6f8IaRQY7M-unsplash-1920.jpg" alt="" class="img-fluid br-frontpage-img frontpage-height">
+      </div>
+      <div class="col-12 col-lg-6">
+      <div class=" mx-0 px-0 vh-100 mt-4 mt-md-0">
+      <div>
+        <div class="text-center d-none d-lg-block">
+          <img src="assets/logo/HikeBuddy-logos_transparent.png" alt="" class="img-fluid">
+        </div>
+        <h1 class="text-center pt-4 pt-lg-2 text-frontpage font-fm-mulish fw-bold fs-2-rem-lg-2-5-rem">Welcome back</h1>
+        <h2 class="text-center pt-2 text-frontpage fs-1rem-lg-1-25rem font-fm-mulish">Login to your account</h2>
+      </div>
+      <form action="/profile/" method="get" class="w-75 m-auto">
+        <div class="pt-3 ">
+          <label for="user-email" class="form-label text-frontpage font-fm-mulish fs-0-75-rem  mb-lg-0">Email address</label>
+          <input type="email" class="form-control" id="user-email" required>
```

Comment:
in the validation ,it's missing the pattern, Oliver has shown us how to do that in a video

Reply:
I am aware of that, and it was intentional as I am not limiting my site for specific domains etc. I am assuming we will keep working on this page further, so I will add JS validation when that time comes. HTML v5 offers quite good basic validation without regex as well.